__tests__/integration/streaming_e2e.test.ts

```diff
@@ -32,13 +31,27 @@ describe('streaming e2e', () => {
       __dirname,
       './fixtures/streaming_e2e_data.ndjson'
     )
-
+    const readStream = fs.createReadStream(filename)
+    const jsonTransform = new Transform({
+      transform(data: Buffer, encoding, callback) {
+        data
+          .toString(encoding)
+          .split('\n')
+          .forEach((line) => {
+            if (!line.length) {
+              return
+            } else {
+              const json = JSON.parse(line)
```

Comment:
I think we should support already serialized data somehow. It looks strange to parse it here and then stringify it again in the library.

Reply:
Let's insert data in `CSV` but query in `JSONCompactEachRow`? It doesn't change the logic much but simplifies the example